### DISCUSSION TITLE
libgfortran error Cartesian Controller

### DISCUSSION THREAD

**Discussion by robertorovella91** [robotology/community#307] [Q&A] [answered]
Hi everyone, i try to use ICub with cartesian controller but when i run  " iKinCartesianSolver --context simCartesianControl --part right_arm" after "yarprobotinterface --context simCartesianControl " the following error occur:
" iKinCartesianSolver: error while loading shared libraries: libgfortran.so.4: cannot open shared object file: No such file or directory "
How can i solve ? 
Thanks in advance.

**Comment by robertorovella91** [comment, marked as answer]
I solve the problem by setting the path in ccmake.

**Comment by pattacini** [comment]
Self-closing issues are those we like the most 😉
